### DISCUSSION TITLE
G Code Fail

### DISCUSSION THREAD

**Discussion by DavideCattaneo97** [OrcaSlicer/OrcaSlicer#2206] [Q&A]
Hi everyone, I have a FlSun SR modified with klipper and I want to start using orca slicer. Does anyone have the start gcode and the end gcode to provide me because once I press the slice button I get this error.
![Immagine 2023-09-23 104201](https://github.com/SoftFever/OrcaSlicer/assets/135587271/8703be4f-2454-4e1f-b1a5-5971df3d64c2)

**Comment by More0than0happy** [comment]
You can try changing {} to [] and that should solve the problem.
If that doesn't work out, you can try my GCODE.
But I am also a newbie, and there is no guarantee that my GCODE is completely correct.
start g-code：
M140 S[first_layer_bed_temperature]
G90
SET_GCODE_OFFSET Z=0.0
G28
G1 Z5 F3000
G1 Z0.15 F300
M190 S[first_layer_bed_temperature]
M109 S[first_layer_temperature]

end g-code：
M140 S0
M104 S0
M106 S0
G91
G1 X-2 Y-2 E-3 F300
G1 Z10 F3000
G90
G28 X Y
M84

I hope my g-code can help you.
